lang/cs/Org.Apache.REEF.Network/Group/Task/IOperatorTopology.cs

```diff
@@ -79,22 +80,24 @@ public interface IOperatorTopology<T>
         /// <summary>
         /// Receive an incoming message from the parent Task.
         /// </summary>
+        /// <param name="cancellationSource">The cancellation token for the data reading operation cancellation</param>
         /// <returns>The parent Task's message</returns>
-        T ReceiveFromParent();
+        T ReceiveFromParent(CancellationTokenSource cancellationSource = null);
```

Comment:
Same as above regarding removing null in future? Moreover, shouldn't this class be internal?